### ISSUE TITLE
script logic mistake

### ISSUE THREAD

**Issue by SilentWarrior1** [postal9191/Cosmo_Miner#2]
Greetings again. Seems something really wrong with the script:
```
Осталось спинов на сегодня 41
Нет доступных спинов. Ожидание 5+ минут перед перезапуском цикла.
Ждем до 2024-09-13 07:05:26.404000+00:00 по UTC для клейма
```
as you can see, i have 41 spins, but... >>script thinks i have no one.

**Comment by postal9191** [comment]
Yes, I understood that you have to manually view 1 ad once a day. Then the whole logic starts working until all attempts are used up.

**Comment by postal9191** [comment]
I can't write a code so that he watches the ad once a day. Maybe I'll do it later. But I have no problem going in and watching the ad once.

**Comment by postal9191** [comment]
![image](https://github.com/user-attachments/assets/cc21e092-1e98-485a-a23c-1507d0ed592e)
![image](https://github.com/user-attachments/assets/9f936705-984d-40cd-8da7-5bbcfbbda497)

The game also has a built-in feature that you can't spin the reel more than 42 times


print('Осталось спинов на сегодня', maxSpins - todaySpinsCount, )
This is server-side protection to prevent advertising abuse
